src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -75,7 +74,7 @@ object Run  {
 
       // Update the database state:
       val newBackendInfo = JesCallBackendInfo(db.CallStatus(ExecutionStatus.Running.toString), Option(JesId(run.jesId)), Option(JesStatus(currentStatus.toString)))
-      dataAccess.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
+      DataAccess.instance.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
```

Comment:
Personally I preferred dataAccess.instance.blahBlahBlah to DataAccess().blahBlahBlah since the former is clear that we're using a singleton instance whereas the latter looks like its creating a new one each time.